pkg/kubelet/images/image_manager_test.go

```diff
@@ -312,3 +313,71 @@ func TestPullAndListImageWithPodAnnotations(t *testing.T) {
 		assert.Equal(t, expectedAnnotations, image.Spec.Annotations, "image spec annotations")
 	})
 }
+
+func TestMaxParallelImagePullLimit(t *testing.T) {
```

Comment:
You can try to run the test with stress to make sure there is no flakes.

Reply:
Can you elaborate on what kind of stress? Or just running the test repeatedly?

Reply:
`go install golang.org/x/tools/cmd/stress@latest`
```
   go test -c -race ./pkg/kubelet/images  -run ^TestMaxParallelImagePullLimit$\n
   /Users/pacoxu/go/bin/stress -p 4 ./images.test
 ```
 for instance.

Reply:
Uh thanks. Ran the test manually:

```
ruiwen@ruiwen:~/ruiwen-zhao/kubernetes$    /usr/local/google/home/ruiwen/go/bin/stress -p 4 ./images.test
...
3m0s: 224 runs so far, 0 failures
```